modules/javafx.graphics/src/test/java/test/javafx/stage/WindowTest.java

```diff
@@ -209,7 +219,7 @@ public void testSetSizeAfterSizeToScene() {
     }
 
     @Test
-    public void testSizeToSceneBeforeShowing() {
+    public void testSizeToSceneBeforeEachShowing() {
```

Comment:
this looks like unrelated change:
- why was the name changed?
- why change this method and not testSizeToSceneAfterShowing?

Reply:
Oops, it is unintentional yeah.

Most of this patch was done with multi-caret editing to tackle multiple spots affected in the same way:
- Mark affected spot
- Ask my IDE to find and put a caret on all other occurrences of selected text
- Edit them all at once

Being in the rhythm of changing imports and `@Before` to `@BeforeEach` I probably unintentionally selected and changed this occurrence of the word "Before". I'll revert this.